cmd/antrea-agent/util_test.go

```diff
@@ -0,0 +1,115 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetAvailableNodePortAddresses(t *testing.T) {
+	testCases := []struct {
+		name                        string
+		nodePortAddressesFromConfig []string
+		excludeDevices              []string
+	}{
+		{
+			name:                        "nodePortAddressesFromConfig equals 0",
+			nodePortAddressesFromConfig: []string{},
+			excludeDevices:              []string{"antrea-egress0", "antrea-ingress0", "kube-ipvs0", "antrea-gw0"},
+		},
+		{
+			name:                        "nodePortAddressesFromConfig not equals 0",
+			nodePortAddressesFromConfig: []string{"1.2.3.0/24", "1.2.3.4/32"},
+			excludeDevices:              []string{"antrea-egress0", "antrea-ingress0", "kube-ipvs0", "antrea-gw0"},
+		},
+	}
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			_, _, goterror := getAvailableNodePortAddresses(tc.nodePortAddressesFromConfig, tc.excludeDevices)
+			assert.NoError(t, goterror)
+		})
+	}
+}
+
+func TestParsePortRange(t *testing.T) {
+	testCases := []struct {
+		//for _, tc := range []struct {
+		name          string
+		portRangeStr  string
+		expectedstart int
+		expectedend   int
+		expectederr   string
+	}{
+		{
+			name:          "wrong portrange format",
+			portRangeStr:  "200 300",
+			expectedstart: 0,
+			expectedend:   0,
+			expectederr:   fmt.Sprintf("wrong port range format: 200 300"),
+		},
+		{
+
+			name:          "length of portrange string not equals to 2 first condition",
+			portRangeStr:  "6100-6200 hello",
+			expectedstart: 0,
+			expectedend:   0,
+			expectederr:   fmt.Sprintf("strconv.Atoi: parsing \"6200 hello\": invalid syntax"),
+		},
+		{
+			name:          "length of portrange string not equals to 2 second condition",
+			portRangeStr:  "6110-7000-8000",
+			expectedstart: 0,
+			expectedend:   0,
+			expectederr:   fmt.Sprintf("wrong port range format: 6110-7000-8000"),
+		},
+		{
+			name:          "wrong portrange type value for start",
+			portRangeStr:  "worng-6200",
+			expectedstart: 0,
+			expectedend:   0,
```

Comment:
You can remove these assignment, default value is 0. Please check other occurrences and remove them.